### ISSUE TITLE
Armature Import Issue, Wont work with Blend files or GLTF

### ISSUE THREAD

**Issue by fase45** [godotengine/godot#69174]
### Godot version

4.0 beta1

### System information

Windows 10 Pro, GLES3 Intel HD Graphics, Blender 3.3

### Issue description

Every time I Import from Blender to Godot no matter what format GLTF/.Blend/FBX the Skeleton3D is messed up. I tried importing animation without armature from Blender and it works fine. When I add armature I have a track for animation in Godot and all the nodes are there, but I can't play the Animation and the Skeleton is incorrect scale or I can't even see it for some reason. 

I don't know if there's settings I need to change or if it's my system, but all file formats are producing the same result and I can't even get any rigged/animated models working because of it.

### Steps to reproduce

1. Create a Blender3D Rig and Animation then export GLTF or just save to .blend file in the Godot project folder. 
2. Open Godot and import Blender3D Rig and Animation.
3. Try to play it.

### Minimal reproduction project

[Import Test.zip](https://github.com/godotengine/godot/files/10094556/Import.Test.zip)

**Comment by fire** [comment]
Pain points.

1. your blend file has a skeleton at the root. Godot has some trouble with this.
2. your skeleton has three roots
3. make the two roots be parented with the first bone
4. then, make sure the godot engine imports influences set to be 4 influences

![image](https://user-images.githubusercontent.com/32321/204073261-1a09ce7a-20ba-4c6c-b09a-42730ebeb15a.png)


[Import Test.zip](https://github.com/godotengine/godot/files/10095707/Impot.Test.zip) Here's an example.

**Comment by fire** [comment]
Here is a video of the project I uploaded.

https://user-images.githubusercontent.com/32321/204073281-b6ab54c9-3f11-419d-9376-f3f714897278.mp4

**Comment by fase45** [comment]
Thanks for your help @fire . Unfortunately, I tried with another very simple Model and still  get the same thing. I used a simple cube and used Armeture. Mad an Empty the scene root, but still no animation and  the bones are overlapping each other.

**Comment by fase45** [comment]
![Capture](https://user-images.githubusercontent.com/14103171/204092357-b544b057-ef84-47a6-b304-e6c99407179f.PNG)
![Capture2](https://user-images.githubusercontent.com/14103171/204092359-10923cae-b03b-42b1-be4c-5501177d8970.PNG)

**Comment by fase45** [comment]
It seemed to have worked on your system can you try to post your project again and I'll try to open it in my system. I don't know if it's OpenGL or something else. But I made one root bone and then also set to 4 influences, changed the scene root to something besides a skeleton.

**Comment by fire** [comment]
[Import Test Renamed.zip](https://github.com/godotengine/godot/files/10097522/Import.Test.Renamed.zip)

Try this one

**Comment by barrrettt** [comment]
I think my case is related. 4.0beta6 blender 3.3.1
blender
![image](https://user-images.githubusercontent.com/47840319/204863302-1c570d8a-c6c0-4518-ba61-3cbcdf1d122a.png)
godot
![image](https://user-images.githubusercontent.com/47840319/204863438-a9748ff7-1678-4cdd-b54b-62fb6613e343.png)

surprisingly everything works well, animations, deformations and positions...

but the skeleton and the meshes are rotated 90º

**Comment by Zireael07** [comment]
@barrrettt What is your up and forward axis set to in Blender?

**Comment by barrrettt** [comment]
![image](https://user-images.githubusercontent.com/47840319/204912239-d2443cef-19fd-4d6d-8cb8-8fec28214521.png)

**Comment by barrrettt** [comment]
everything works fine. look the bone attackment:
![image](https://user-images.githubusercontent.com/47840319/204914732-6f240bca-6501-4d89-a20b-e5738ca9d79b.png)

it may be an error when the gizmos cube is render:
![image](https://user-images.githubusercontent.com/47840319/204914322-67e0b16a-b7fe-4fcd-9a30-75449ed025b5.png)

**Comment by barrrettt** [comment]
Thinking maybe this is not the right place for my issue. if you need more information, please ask. i can't think of anything else that would be helpful. Maybe just skeletons and meshes have different functions for the change of base?

**Comment by Zireael07** [comment]
@barrrettt Yes, likely open a new issue, though it's almost certainly user error on your end (as evidenced by the screenshots, Blender uses Z as up and Godot uses Y as up, thus explaining the 90 deg rotation).

You should have an axis setting when exporting in Blender, try fixing it to conform to the Godot convention

**Comment by Zireael07** [comment]
> I say this because I have the blender defaults

This is literally the core of your problem. Blender defaults differ from Godot convention

**Comment by barrrettt** [comment]
In that case I would suggest to put some kind of warning or option in the blender file importer.

**Comment by barrrettt** [comment]
I cannot reproduce this behavior. it must be a local problem. good

**Comment by fase45** [comment]
@fire Send me a video of you fixing the problem because as it stands for me i can see the skeleton but it still wont animate or pose. I used all the settings you mentioned and something is still wrong. The FPSARMS.tscn you sent says it has corrupted and won't work. Is anyone else running into this issue. I don't get why it's so hard for me to implement one of the key aspects of game animation. Please someone fix this. I can't even use Godot 4 at this point(just been playing with the animation settings since downloaded it). I've been in 3.5 because I am stuck. I don't know if it's OpenGL or what.



![GODOT 4 ISSUE](https://user-images.githubusercontent.com/14103171/206606266-19f1b4b2-ee98-473b-a55d-661047f2cc52.PNG)

**Comment by fase45** [comment]
Thank you
Hopefully someone can tell me whats going on

**Comment by fase45** [comment]
For anyone wondering here's a simple animation and armature same problem optimize is even off


https://user-images.githubusercontent.com/14103171/206635703-9aa9ffd8-98a7-4196-b3b0-8a4828a43f99.mp4

**Comment by fase45** [comment]
I just got Godot  4 beta 8 and it works!!! Thank you Everyone
